src/snyk/snykCode/views/issueTreeProvider.ts

```diff
@@ -131,6 +136,7 @@ export class IssueTreeProvider extends AnalysisTreeNodeProvder {
         });
         review.push(file);
       });
+    }
```

Comment:
Please remove commented code - we do have git after all 😃.

Reply:
✅